pypeit/flatfield.py

```diff
@@ -235,14 +235,16 @@ def get_spat_bsplines(self, frametype='illum'):
                 msgs.warn("pixelflat has no spatial bspline fit - using the illumflat")
                 return self.illumflat_spat_bsplines
 
-    def get_pixelflat(self):
-        return self.pixelflat_norm
+    # TODO: Why are these 3 functions necessary?  I've removed them and edited
```

Comment:
I forget..

Reply:
I've removed their use from the code, and I've now removed the commented code.